database/dump.sql

```diff
@@ -350025,154 +350026,154 @@ CREATE TABLE IF NOT EXISTS `page_repository` (
-(21398, 1, 1, NULL, 27030),
-(21406, 1, 1, NULL, 33870),
-(21408, 1, 1, NULL, 33085),
-(21413, 1, 1, NULL, 27031),
-(21421, 1, 1, NULL, 31780),
-(21423, 1, 1, NULL, 24563),
-(21427, 1, 1, NULL, 21428),
-(21429, 1, 1, NULL, 31183),
-(21431, 1, 1, NULL, 32784),
-(21433, 1, 1, NULL, 29289),
-(21435, 1, 1, NULL, 23031),
-(21437, 1, 1, NULL, 35467),
-(21439, 1, 1, NULL, 35308),
-(21456, 1, 1, NULL, 35461),
-(21468, 1, 1, NULL, 34048),
-(21470, 1, 1, NULL, 32267),
-(21472, 1, 1, NULL, 32792),
-(21475, 1, 1, NULL, 21476),
-(21511, 1, 1, NULL, 31777),
-(21526, 1, 1, NULL, 31769),
-(21538, 1, 1, NULL, 26924),
-(21541, 1, 1, 21546, 35151),
-(21543, 1, 1, 21546, 21544),
-(21549, 1, 1, NULL, 31774),
-(21551, 1, 1, NULL, 24472),
-(21553, 1, 1, NULL, 21554),
-(21555, 1, 1, NULL, 29301),
-(21557, 1, 1, NULL, 31778),
-(21559, 1, 1, NULL, 23832),
-(21561, 1, 1, NULL, 24453),
-(21563, 1, 1, NULL, 35411),
-(21565, 1, 1, NULL, 33953),
-(21567, 1, 1, NULL, 32537),
-(21570, 1, 1, 23842, 34543),
-(21654, 1, 1, NULL, 24352),
-(21657, 1, 1, NULL, 33833),
-(21933, 1, 1, NULL, 24494),
-(22886, 1, 1, NULL, 22887),
-(22964, 1, 1, NULL, 24353),
-(23111, 1, 1, NULL, 23230),
-(23320, 1, 1, NULL, 30098),
-(23439, 1, 1, NULL, 24723),
-(23534, 1, 1, NULL, 34751),
-(23576, 1, 1, NULL, 23587),
-(23579, 2, 1, 23599, 23583),
-(23580, 2, 1, 23599, 23602),
-(23591, 2, 9, NULL, 32995),
-(23711, 2, 1, NULL, 23714),
-(23720, 2, 1, NULL, 23753),
-(23727, 2, 1, NULL, 32962),
-(23950, 1, 1, NULL, 33031),
-(24214, 1, 1, 23842, 24722),
-(24706, 1, 1, NULL, 33892),
-(24711, 1, 1, NULL, 34454),
-(24806, 1, 1, 23842, 32994),
-(24887, 1, 1, NULL, 35470),
-(25017, 1, 1, 25016, 34402),
-(25019, 1, 1, 25016, 27116),
-(25063, 1, 1, 21396, 25065),
-(25074, 1, 1, 21396, NULL),
-(25079, 2, 1, NULL, 25081),
-(25082, 2, 9, NULL, 32879),
-(25294, 1, 1, 21546, 34455),
-(25363, 1, 1, 21546, 35384),
-(25373, 1, 1, 21546, 25374),
-(25598, 1, 1, NULL, 25654),
-(25713, 1, 1, NULL, 34708),
-(25985, 1, 1, NULL, 26298),
-(26087, 1, 1, 25341, 27147),
-(26089, 1, 1, 25341, 26434),
-(26095, 1, 1, 21396, 34136),
-(26245, 1, 1, 21396, 35440),
-(26453, 1, 1, 21396, 33136),
-(26473, 1, 1, NULL, 26512),
-(26524, 1, 1, NULL, 26525),
-(26542, 1, 1, 25016, 27319),
-(26544, 1, 1, 25016, 27323),
-(26546, 1, 1, 25016, 32810),
-(26592, 1, 1, 25016, 27316),
-(26633, 1, 1, 25341, 27504),
-(26639, 1, 1, 20829, 26674),
-(26874, 1, 1, NULL, 26875),
-(26880, 1, 1, NULL, 26881),
-(27203, 1, 1, 25016, 32494),
-(27421, 1, 1, 21396, 31712),
-(27469, 2, 9, NULL, 27470),
-(27472, 1, 1, NULL, 27631),
-(31996, 1, 1, 25341, 32175),
-(32840, 2, 9, NULL, 32846),
-(32875, 1, 1, 20541, 33367),
-(32966, 2, 9, NULL, 32967),
-(35093, 1, 1, 25333, 35376),
-(35096, 1, 1, 25333, 35378),
-(35098, 1, 1, 25333, 35381),
-(35100, 1, 1, 25333, 35383),
-(35152, 1, 1, 25333, 35373);
+INSERT INTO `page_repository` (`id`, `instance_id`, `license_id`, `forum_id`, `current_revision_id`, `discussions_enabled`) VALUES
+(16256, 1, 1, NULL, 16285, 0),
```

Comment:
there's actually no need for this because `discussions_enabled boolean NOT NULL DEFAULT '0'` is `0` per default (when left empty) :)